core/vm/operations_verkle.go

```diff
@@ -74,14 +74,15 @@ func gasExtCodeHash4762(evm *EVM, contract *Contract, stack *Stack, mem *Memory,
 
 func makeCallVariantGasEIP4762(oldCalculator gasFunc) gasFunc {
 	return func(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
+		address := stack.Back(1).Bytes20()
 		gas, err := oldCalculator(evm, contract, stack, mem, memorySize)
 		if err != nil {
 			return 0, err
 		}
-		if _, isPrecompile := evm.precompile(contract.Address()); isPrecompile {
+		if _, isPrecompile := evm.precompile(address); isPrecompile {
 			return gas, nil
 		}
-		wgas := evm.Accesses.TouchAndChargeMessageCall(contract.Address().Bytes())
+		wgas := evm.Accesses.TouchAndChargeMessageCall(address[:])
```

Comment:
So this might be a problem for `DELEGATECALL` as I feel that only the code of the target account should be touched. This needs to be debated during interop.